ckanext/xroad_integration/logic/action.py

```diff
@@ -818,45 +819,6 @@ def fetch_distinct_service_stats(context, data_dict):
         return {"success": False, "message": "Fetching distinct service statistics failed."}
 
 
-def fetch_list_errors(context, data_dict):
-    toolkit.check_access('fetch_list_errors', context)
-
-    organization = data_dict.get('organization')
-    page = 0
-    if "page" in data_dict and data_dict.get('page') is not None:
-        page = data_dict.get('page')
-    limit = DEFAULT_LIST_ERRORS_PAGE_LIMIT
-    if "limit" in data_dict and data_dict.get('limit') is not None:
-        limit = data_dict.get('limit')
-    if not organization:
-        return []
-    xroad_id = organization.split('.')
-    xroad_instance = xroad_id[0] if len(xroad_id) == 3 else ''
-    member_class = xroad_id[1] if len(xroad_id) == 3 else ''
-    member_code = xroad_id[2] if len(xroad_id) == 3 else ''
-
-    days = DEFAULT_LIST_ERRORS_HISTORY_IN_DAYS
-
-    log.info("Fetching X-Road errors for the last %s days" % days)
-
-    try:
-        pagination = {"page": str(page), "limit": str(limit)}
-        error_data = xroad_catalog_query('listErrors', [xroad_instance, member_class, member_code, str(days)], pagination=pagination).json()
-
-        if error_data is None:
-            log.warn("Calling listErrors failed!")
-            return {'success': False, 'message': 'Calling listErrors failed!'}
-        elif "errorLogList" not in error_data:
-            return []
-
-        return error_data
-
-    except ConnectionError as e:
-        log.warn("Calling listErrors failed!")
-        log.info(e)
-        return {"success": False, "message": "Fetching errors failed."}
-
-
```

Comment:
How the errors are now fetched from xroad catalog ?

Reply:
This was for the listErrors endpoint, the previous error fetching is still done in "def fetch_xroad_errors(context, data_dict)"